tests/server/api/api_test.go

```diff
@@ -945,9 +945,9 @@ func TestRemovingProgress(t *testing.T) {
 	// store 2: (30-10)/(30+40) ~= 0.285
 	re.Equal("0.29", fmt.Sprintf("%.2f", p.Progress))
 	// store 2: 20/10s = 2
-	re.Equal(2.0, p.CurrentSpeed)
+	re.LessOrEqual(2.0, p.CurrentSpeed)
 	// store 2: (10+40)/2 = 25s
-	re.Equal(25.0, p.LeftSeconds)
+	re.GreaterOrEqual(25.0, p.LeftSeconds)
```

Comment:
Will it change the behavior of test?

Reply:
No, the reason is that checkStores is executed more than once.